vsintegration/src/FSharp.ProjectSystem.FSharp/Project.fs

```diff
@@ -355,6 +355,14 @@ namespace rec Microsoft.VisualStudio.FSharp.ProjectSystem
                 pIdIco <- 400u
                 VSConstants.S_OK
 
+            override this.Dispose(disposing) =
```

Comment:
We guard revoking component by `componentId` and `mgr` so I think double revoking should not happen.